StateMachine.mmd

```diff
@@ -44,7 +44,6 @@ stateDiagram-v2
   
   state INTAKE_ALGAE_WITH_CORAL_GROUP {
       INTAKE_ALGAE_WITH_CORAL_GROUND
```

Comment:
We should probably rename these states (and coral ones) like this to make it less confusing
```suggestion
      INTAKE_ALGAE_GROUND_WITH_CORAL
```